dcos/dcos-ui
# feat(PackageDetailTab): implement package version dropdown

This commit includes the work for epic DCOS_OSS-1528 where as a user now
is able to select a version in package detail page.

What is included:
- package detail page UI dropdown element
- Actions to retrieve data for package versions

How to test:
- Go to catalog page click on a package
- Notice the dropdown at the top where the package name is
- When choosing a version the content should change (if there is any difference) to the chosen version
- When deployed the version selected will be installed

Closes DCOS_OSS-1560 DCOS_OSS-1559 DCOS_OSS-1558

**before:**
<img width="1025" alt="screen shot 2017-08-21 at 2 55 25 pm" src="https://user-images.githubusercontent.com/549394/29540017-5bc3c1ce-8681-11e7-8bf1-8405fb817a50.png">

**after:**
<img width="1059" alt="screen shot 2017-08-21 at 2 58 39 pm" src="https://user-images.githubusercontent.com/549394/29540030-66af0c7e-8681-11e7-9703-4628955ed7e1.png">


**Checklist**
